Styles/style.css

```diff
@@ -170,22 +185,21 @@ body {
       transform: rotate(360deg);
     }
   }
-
   .sidenav {
     overflow-y: scroll;
     margin-left: -289px;
     transition: ease-in-out 500ms;
     height: 100%;
-    background: rgba(45, 50, 59, 1);
+    background-color: #fff;
     box-shadow: 0 8px 10px 0 rgba(53, 186, 226, 0.37);
-    /* backdrop-filter: blur(20px);
-    -webkit-backdrop-filter: blur(20px); */
+    backdrop-filter: blur(20px);
+    -webkit-backdrop-filter: blur(20px);
     border-radius: 10px;
     border: 1px solid rgba(255, 255, 255, 0.18);
     z-index: 1;
```

Comment:
looking at z-index:1 reminded me of kushagra saying z-index should be used in variables and the variables should be kept at a single file,this way when we want some ui to overwrite we dont have to go like z-index:100000